ci/install_gcovr.sh

```diff
@@ -1,14 +1,32 @@
 #!/bin/bash
 #
-# Install gcovr-4.1 as the default gcovr
+# Prerequisites:
+#   git
+#   python
+#   pip
 KERNEL_NAME=$(uname -s)
 
+# MacOS Support
 if [ "${KERNEL_NAME}" == "Darwin" ]; then
-  brew install gcovr || exit 1
+  brew install git python || exit 1
+
+# Debian Support
 else
   (apt-get -qq update &&
-  apt-get -qq install python-pip &&
-  pip install gcovr==4.1) || exit 2
+  apt-get -qq install git python-pip) || exit 1
+fi
+
+# Install custom `gcovr` to support Coveralls output
+pushd ~
+git clone https://github.com/zfields/gcovr.git -b coveralls
```

Comment:
I think it's normally best to keep device-os dependencies in a particle repo when we can. I realize your PR may be merged into the main gcovr repo, so we'll leave this as is for now. I've set myself a reminder to check back about this in 1 month.